test/parallel/test-common-debuglog-dummy.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const common = require('../common');
+
+// argv[0] is the execPath, argv[1] is the test script filename
+common.debuglog.apply(null, process.argv.slice(2));
```

Comment:
This should either be inlined into the new `test/parallel/test-common-debuglog.js` (see [`test/parallel/test-child-process-spawnsync-env.js`](https://github.com/nodejs/node/blob/c7962dcba46f44b888686e6a76bcc6fef2eb7760/test/parallel/test-child-process-spawnsync-env.js) for how tests that spawn other processes are typically written) or this file should be moved to `test/fixtures`.

Reply:
will move it to `test/fixtures`, since the same file will need to be invoked with on different child processes... Inlining it might make it needlessly complex, to test for different values of `NODE_DEBUG`